challenges.yml

```diff
@@ -0,0 +1,37 @@
+extending_instancedynamicdata:
+  question: |
+    Suppose we have a vue component that looks like this:
+
+    ```javascript
+    const app = new Vue({
+      el: '#app',
+      template: '<span class="title">{{ title }}</span>
+    });
+    ```
+
+    and the `title` is a data property that resides in that same component,
+    how would you go about initializing this data?
+  explanation: |
+    That's right! `data` is a `function` that returns an object. The properties
+    in this object will be accessible from the template! The function shorthand
```

Comment:
This might be a good spot to explain why it needs to be a function (in some light detail(: because it allows Vue to keep different data for each instance of your Vue app, in case you have multiple instances.

Reply:
This is not really the case (https://vuejs.org/v2/guide/instance.html#Data-and-Methods) Data can be just an object. Vue will process this object every time it instantiates the App and assign it to the instance (remember these are options we pass here, not the real Vue component that runs in the Browser)